WordPress/Classes/ViewRelated/Reader/Detail/ReaderDetailViewController.swift

```diff
@@ -144,7 +144,10 @@ class ReaderDetailViewController: UIViewController, ReaderDetailView {
     /// allowing Reader Detail to use a blue navbar.
     var useCompatibilityMode: Bool {
         // Use compatibility mode if not presented within the Reader
-        return WPTabBarController.sharedInstance().readerNavigationController.viewControllers.contains(self) == false
+        guard let tabBarController = RootViewControllerCoordinator.sharedPresenter.rootViewController as? WPTabBarController else {
```

Comment:
Same comment as mentioned above with accessing `WPTabBarController?` directly

Reply:
Done in d8978105f0e302436aa4afe7ffea5f4123c76ebf 👍